exec/java-exec/src/main/java/org/apache/drill/exec/ExecConstants.java

```diff
@@ -421,13 +422,13 @@
    * such as changing system options.
    */
   String ADMIN_USERS_KEY = "security.admin.users";
-  StringValidator ADMIN_USERS_VALIDATOR = new StringValidator(ADMIN_USERS_KEY, true);
+  StringValidator ADMIN_USERS_VALIDATOR = new StringValidator(ADMIN_USERS_KEY, new OptionValue.MetaData(true, false));
```

Comment:
We are adding a distinction between runtime options that the user can set vs. those that the user cannot set? This is a very bizarre distinction as runtime options are only for the user...

Actually, these are options that are "public" vs. those that exist, but are generally "hidden", right?

Another very useful addition, while we're in the code, is between the allowed "scope" of a setting. Today, all options can be set at the system and session (per user) level. But some options, such as the memory per query, or enabling queues, really only make sense at the system level. So, part of the new metadata should be to specify "session-only" options and forbid setting such options at the session level.